adapters/consumable/consumable.go

```diff
@@ -170,6 +171,15 @@ func (a *ConsumableAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *
 		} else {
 			gdpr.Consent = extUser.Consent
 			body.GDPR = &gdpr
+
+			for i := 0; i < len(extUser.Eids); i++ {
+				if len(extUser.Eids[i].UIDs) > 0 {
+					if extUser.Eids[i].UIDs[0].ID != "" {
+						body.User.Eids = extUser.Eids
+					}
+					break
+				}
+			}
```

Comment:
Might be better expressed as a method, something like:

```
if eidsNotEmpty(extUser.Eids) {
	body.User.Eids = extUser.Eids
}

...

func eidsNotEmpty(eids []openrtb2.EID) bool {
	for i := 0; i < len(eids); i++ {
		if len(eids[i].UIDs) > 0 && eids[i].UIDs[0].ID != "" {
			return true
		}
	}
	return false
}
```

Reply:
updated, thanks for the suggestion